source/implode.rst

```diff
@@ -44,7 +57,7 @@ Keyboard Shortcuts
 
 On an XO laptop, the game can also be played via the game keys and directional pad:
 
-* Use the directional pad to move the selection cursor.
+* Use the directional pad on an XO laptop to move the selection cursor.
```

Comment:
Redundant text; the list introducer already says on an XO laptop.

Reply:
removed. thanks